test/unit/wallet.test.ts

```diff
@@ -41,4 +51,58 @@ describe('Alchemy-Ethers Wallet', () => {
     const rawTx = await wallet.signTransaction(transaction);
     expect(rawTx).toEqual(expectedRawTx);
   });
+
+  it('Matches functionality of Ethers Signer', async () => {
+    const alchProvider = await alchemy.config.getProvider();
+
+    const alchWallet = new Wallet(privateKey, alchemy);
+    const ethersWallet = new EthersWallet(privateKey, alchProvider);
+    const blockTag = 15000000;
+
+    expect(await alchWallet.getBalance(blockTag)).toEqual(
+      await ethersWallet.getBalance(blockTag)
+    );
```

Comment:
This is fine, but fyi you can also write these as:
```ts
await expect(alchWallet.getBalance(blockTag)).resolves.toEqual(
  await ethersWallet.getBalance(blocktag)
);
```
It's more verbose, but it makes a better error message in the case where the promise fails.